.github/workflows/stable-cicd.yaml

```diff
@@ -73,6 +73,31 @@ jobs:
                     pypi_username: ${{secrets.PYPI_USERNAME}}
                     pypi_password: ${{secrets.PYPI_PASSWORD}}
                     ADMIN_GITHUB_TOKEN: ${{secrets.ADMIN_GITHUB_TOKEN}}
+            -
+                name: 🫙 Docker Image Tag Determination
+                id: docker_tag_determination
+                run: echo "::set-output name=image_tag::$(cat ./src/pds/registrysweepers/VERSION.txt)"
```

Comment:
FYI, we'll need to move `::set-output` → `$GITHUB_OUTPUT` in the near future

Reply:
Got a reminder for this and at least one other repository - need to enumerate affected repos